### ISSUE TITLE
only available in english

### ISSUE THREAD

**Issue by Silimilikim** [graywolf336/RocketChatApps#12]
I would like to use this app in french. Translating in different languages is a huge work i know. But at least could we have the possiblity to customize the out-of-office messages "is currently out of office, however they left the following message:" ?

**Comment by graywolf336** [comment]
Would you be able to translate it into French? @Silimilikim

**Comment by Silimilikim** [comment]
yes !

**Comment by graywolf336** [comment]
@Silimilikim if you're still interested, I just merged a pull request to add Italian and will happily accept French translations!

**Comment by Silimilikim** [comment]
@graywolf336 Yes, still willing to help. Tell me what you need me to translate.

**Comment by graywolf336** [comment]
@Silimilikim all of the files which are in the files named `i18n/en.json` need to be copied and named `i18n/fr.json` I believe.

**Comment by ankar84** [comment]
Hey @graywolf336 
Please add Russian support https://github.com/graywolf336/RocketChatApps/pull/26
Thank you!
